tests/iio_common.c

```diff
@@ -126,6 +126,23 @@ struct iio_context * autodetect_context(bool rtn, const char * name, const char
 	return ctx;
 }
 
+ssize_t iio_device_enable_channel(const struct iio_device *dev, const char * channel, bool type)
```

Comment:
Function return type could be just `int`.

Reply:
I think that caused an issue/warning on Windows - will double check...

Reply:
yeah, I didn't see the same warnings this time - changed and squashed.